include/NovelRT/ResourceManagement/ResourceLoader.h

```diff
@@ -31,16 +26,27 @@ namespace NovelRT::ResourceManagement
             return _resourcesRootDirectory;
         }
 
-        [[nodiscard]] inline TextureMetadata LoadTextureFromFile(const std::string& fileName)
-        {
-            return LoadTexture(_resourcesRootDirectory / "Images" /
-                               fileName); // TODO: This should probably either be textures or sprites, not image? Thonk.
-        }
-
-        [[nodiscard]] inline std::vector<uint8_t> LoadShaderSource(const std::string& fileName)
-        {
-            return LoadShaderSourceInternal(_resourcesRootDirectory / "Shaders" / fileName);
-        }
+        /**
+         * @brief Loads texture from file on given path.
+         *
+         * Path can both be relative as well as absolute.
+         * When using relative path it will look in the Resources + Images directory
```

Comment:
```suggestion
         * The path can be either relative or absolute.
         * When using a relative path it will look in the Resources/Images directory.
```